### ISSUE TITLE
spigot-api-1.12-pre5-SNAPSHOT load error

### ISSUE THREAD

**Issue by dordsor21** [St3venAU/ArmorStandTools#12]
https://hastebin.com/amogajigog.vbs

outputted to console on load using spigot-api-1.12-pre5-SNAPSHOT

**Comment by St3venAU** [comment]
AST is not yet updated for 1.12. Keep an eye on this thread for updates: https://www.spigotmc.org/threads/armor-stand-tools.38107/page-14#post-2423107

**Comment by dordsor21** [comment]
Tested https://github.com/St3venAU/ArmorStandTools/pull/13 and should fix.

**Comment by St3venAU** [comment]
Thanks, but the config file handling isn't the only thing that's changed. I've found that a bunch of the NMS field names have changed (as they often do with new MC versions), which is causing errors when using the plugin. I'm in the process of getting them fixed up. Once I've done that and I've tested all the functionality of the plugin in all of the supported spigot versions I will release the update. Cheers.
